hugo/layouts/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-agent: *
+
+Sitemap: https://alpha.cuelang.org/sitemap.xml
```

Comment:
Can we make this coming from the hugo config instead of hardcoded using https://gohugo.io/functions/absurl/ ?